packages/terra-doc-template/src/DocTemplate.jsx

```diff
@@ -17,6 +18,14 @@ const propTypes = {
    * The given component's npm package name.
    */
   packageName: PropTypes.string,
+  /**
+   * The url for the provided package if not in npm. This will be ignored if packageVersion is not set.
+   */
+  packageUrl: PropTypes.string,
```

Comment:
What url do we expect people to provide internally? Will this differ from the component source code URL we already supply on the page? 

What if someone want so to render the badge with a version but doesn't care to supply a link?

Reply:
They might want to point it to an internal source, or they may just want it to loop right back to the same page the badge is found on to simply not have a 404 badge.

If they supply a version without a url, it falls back to passing the version to the normal npm link, pointed at that version of the package. If that doesn't exist, the url prop gives them the option to point it wherever they desire.